app/controllers/user_controller.php

```diff
@@ -3,19 +3,26 @@
 class UserController extends AppController
 {
     CONST PAGE_REGISTER = 'register';
-    CONST PAGE_LOG_IN   = 'log_in';
+    CONST PAGE_LOG_IN   = 'user/log_in';
     CONST PAGE_REGISTER_END = 'register_end';
     CONST PAGE_LOG_IN_END   = 'log_in_end';
+    CONST PAGE_EDIT_PROFILE = 'edit_profile';
+    CONST PAGE_EDIT_PROFILE_END = 'profile_end';
+    CONST PAGE_EDIT_PASSWORD_END = 'password_end';
+    CONST PAGE_PROFILE_NOT_FOUND = 'profile_not_found';
 
     public function register()
     {
         $user = new User();
         $page = Param::get('page_next', self::PAGE_REGISTER);
-        $user->username = Param::get('username');
-        $user->email = Param::get('email');
+        $user->username = trim(Param::get('username'));
+        $user->email = trim(Param::get('email'));
+        $user->firstname = trim(Param::get('firstname'));
+        $user->lastname = trim(Param::get('lastname'));
+        $user->department = trim(Param::get('department'));
```

Comment:
For trim() po, please refer [here](https://github.com/adrianneperedo/board/pull/1#discussion_r25484187)